api/v1alpha3/machineset_types.go

```diff
@@ -201,6 +205,26 @@ type MachineSet struct {
 	Status MachineSetStatus `json:"status,omitempty"`
 }
 
+// GetConditions retrieves all Status Conditions.
+func (m *MachineSet) GetConditions() Conditions {
+	return m.Status.Conditions
+}
+
+// SetConditions sets all Status Conditions.
+func (m *MachineSet) SetConditions(conditions Conditions) {
+	m.Status.Conditions = conditions
+}
+
+// GetConditions retrieves all Status Conditions.
+func (m *MachineSetStatus) GetConditions() Conditions {
+	return m.Conditions
+}
+
+// SetConditions sets all Status Conditions.
+func (m *MachineSetStatus) SetConditions(conditions Conditions) {
+	m.Conditions = conditions
+}
```

Comment:
Are these methods necessary? I believe we only define GetConditions/SetConditions on the top level type in other places.

Reply:
Yeah, I added them in to avoid having to take a copy of the entire MS, but MachineSetStatus is not a valid runtime object so that didn't work (won't compile).  I will remove these.